### ISSUE TITLE
Compiling fails on Xcode 14

### ISSUE THREAD

**Issue by lemonandlime** [klarna/klarna-mobile-sdk-spm#1]
```
KlarnaView.swift:10:8: error build: Failed to build module 'KlarnaMobileSDK'; this SDK is not supported by the compiler (the SDK is built with 'Apple Swift version 5.5.1 (swiftlang-1300.0.31.4 clang-1300.0.29.6)', while this compiler is 'Apple Swift version 5.7 (swiftlang-5.7.0.127.4 clang-1400.0.29.50)'). Please select a toolchain which matches the SDK.

**Comment by polar133** [comment]
Hi @lemonandlime,

Thanks for the information. We just released a new version compiled using the latest stable version of Xcode (14.0.1). Please give it a try and let us know how it goes.

**Comment by polar133** [comment]
We released a new version (2.3.1) that should work. Please, give it a try and let us know how it goes.
